From ae097f242dc2da357fe53473e312aab5ed9badb5 Mon Sep 17 00:00:00 2001
From: "jon.lin" <jon.lin@rock-chips.com>
Date: Mon, 12 Nov 2018 19:08:39 -0800
Subject: [PATCH] ARM: dts: rk312x: sfc frequency reduction

1.It is found that some SPI Nand device run abnormal at 100M frequency in
rk3128 gameplay project

Change-Id: I78f78c7ea5eea5d294d2a195bc68141283c515f3
Signed-off-by: jon.lin <jon.lin@rock-chips.com>
---
 arch/arm/boot/dts/rk312x.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk312x.dtsi b/arch/arm/boot/dts/rk312x.dtsi
index 86acf9e6a4ad..d56329de0d9b 100644
--- a/arch/arm/boot/dts/rk312x.dtsi
+++ b/arch/arm/boot/dts/rk312x.dtsi
@@ -785,7 +785,7 @@
 		clocks = <&cru SCLK_SFC>, <&cru HCLK_SFC>;
 		clock-names = "clk_sfc", "hclk_sfc";
 		assigned-clocks = <&cru SCLK_SFC>;
-		assigned-clock-rates = <100000000>;
+		assigned-clock-rates = <60000000>;
 		status = "disabled";
 	};
 
-- 
2.35.3

